docs/supported.md

```diff
@@ -323,7 +323,9 @@ Macros can also be defined in the KaTeX [rendering options](options.md).
 
 Macros accept up to nine arguments: #1, #2, etc.
 
-`\gdef`, `\xdef`, `\global\def`, `\global\edef`, `\global\let`, and `\global\futurelet` will persist between math expressions.
+<div id="gdef"></div>
+
+`\gdef`, `\xdef`, `\global\def`, `\global\edef`, `\global\let`, and `\global\futurelet` will persist between math expressions. (Exception: macro persistence may be disabled for security reasons.)
```

Comment:
1. For correctness, should this sentence be prefixed by `Macros defined by`?  (It's the macros that persist, not `\gdef` itself.)
2. I would personally reword the second sentence to something like `(assuming this feature has been [properly setup](api.html#persistent-macros) and not disabled for security reasons)`. But up to you whether to reword; I understand you want to keep this simple.

Reply:
Comments picked up. I revised one of them to avoid implying incompetence on the part of the web site administrator.